cpp/src/plasma/store.cc

```diff
@@ -727,7 +728,7 @@ void PlasmaStore::DisconnectClient(int client_fd) {
       sealed_objects[it->first] = it->second.get();
     } else {
       // Abort unsealed object.
-      AbortObject(it->first, client);
+      EraseFromObjectTable(object_id);
```

Comment:
Add a comment here that `EraseFromObjectTable()` is called instead of `AbortObject()` because the latter modifies client->object_ids, or you can just call  `AbortObject()` here and revise the iterator.

Reply:
Thank you. The comment is added.